tools/ci_build/build.py

```diff
@@ -1587,16 +1587,6 @@ def build_python_wheel(
         args = [sys.executable, os.path.join(source_dir, 'setup.py'),
                 'bdist_wheel']
 
-        # We explicitly override the platform tag in the name of the generated build wheel
-        # so that we can install the wheel on Mac OS X versions 10.12+.
-        # Without this explicit override, we will something like this while building on MacOS 10.14 -
-        # [WARNING] MACOSX_DEPLOYMENT_TARGET is set to a lower value (10.12)
-        # than the version on which the Python interpreter was compiled (10.14) and will be ignored.
-        # Since we need to support 10.12+, we explicitly override the platform tag.
-        # See PR #3626 for more details
-        if is_macOS():
-            args += ['-p', 'macosx_10_12_x86_64']
```

Comment:
Similar to the comment above, if we ever use Python with a minimum mac deployment target as 10.15, the platform tag used for the generated wheel will be based off that right ? Should we still keep this overriding logic ?